packages/lib/apps/getEnabledApps.ts

```diff
@@ -2,12 +2,47 @@ import getApps from "@calcom/app-store/utils";
 import type { CredentialData } from "@calcom/app-store/utils";
 import { prisma } from "@calcom/prisma";
 
-const getEnabledApps = async (userCredentials: CredentialData[]) => {
+/**
+ *
+ * @param credentials - Can be user or team credentials
+ * @param filterOnCredentials - Only include apps where credentials are present
+ * @returns A list of enabled app metadata & credentials tied to them
+ */
+const getEnabledApps = async (credentials: CredentialData[], filterOnCredentials?: boolean) => {
+  const filterOnIds = {
+    credentials: {
+      some: {
+        OR: [],
+      },
+    },
+  };
+  if (filterOnCredentials) {
+    const userIds: number[] = [],
+      teamIds: number[] = [];
+
+    for (const credential of credentials) {
+      if (credential.userId) userIds.push(credential.userId);
+      if (credential.teamId) teamIds.push(credential.teamId);
+    }
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    //@ts-ignore
+    if (userIds.length) filterOnIds.credentials.some.OR.push({ userId: { in: userIds } });
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    //@ts-ignore
+    if (teamIds.length) filterOnIds.credentials.some.OR.push({ teamId: { in: teamIds } });
+  }
+
   const enabledApps = await prisma.app.findMany({
-    where: { enabled: true },
+    where: {
+      OR: [
+        { enabled: true, ...(filterOnIds.credentials.some.OR.length && filterOnIds) },
+        // Even if filtering on credentials, everyone has Daily installed
+        { slug: "daily-video", enabled: true },
```

Comment:
Since Daily credentials do not exisit then everyone technically has this app installed.